lib/crowbar/client/command/node/list.rb

```diff
@@ -62,7 +62,7 @@ def headings
           end
 
           def content_from(request)
-            [].tap do |row|
+            result = [].tap do |row|
```

Comment:
Not me, but rubocop rules :)
https://github.com/crowbar/crowbar-client/pull/51#discussion_r49452119